### PR TITLE
submission

### PR DESCRIPTION
**Pull request by ghost**
comfort: 2.75 and movin' on up
completeness: 3.5

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -9,3 +9,36 @@
...
+for (color = 0; color < colors.length; ++color) {
+    console.log("My favorite color is " + colors[color]);
+}
+
+var item
+var instructor = {
+    foam:           "barbasol",
+    blade:          "Harrys",
+    friendship:     "bestship",
+    facialhair:     "instructor"
```
Comment (jshawl) [inline script.js:29]: should be camel case or hyphen

File: script.js [hunk 2]
```diff
@@ -9,3 +9,36 @@
...
+}
+
+
+// found this on stackoverflow, I have a good(not great) idea of what it's doing per line.
```
Comment (jshawl) [inline script.js:33]: should be `console.log(instructor.facialhair)`

File: index.html [hunk 3]
```diff
@@ -11,5 +11,7 @@
     <div id="yellow" class="bulb"></div>
     <div id="green" class="bulb"></div>
   </div>
+    <script src="https://code.jquery.com/jquery-3.1.0.slim.min.js"></script>
+    <script src="script.js"></script>
```
Comment (jshawl) [inline index.html:15]: missing link to stoplight.js

File: stoplight.js [hunk 4]
```diff
@@ -0,0 +1,30 @@
+/*  from the pre pre work before WDI12, 
+    http://codepen.io/mikeflores/pen/xOZWkv?editors=0010
+    copy and pasted the code I worked and I would modify it to work with the elements in the index
+*/
+
+document.getElementById('stopButton').onclick = illuminateRed;
+document.getElementById('slowButton').onclick = illuminateYellow;
+document.getElementById('goButton').onclick = illuminateGreen;
+document.getElementById('clearButton').onclick = clearLights;
```
Comment (jshawl) [inline stoplight.js:9]: none of these ids are in the HTML

File: stoplight.js [hunk 5]
```diff
@@ -0,0 +1,30 @@
+/*  from the pre pre work before WDI12, 
+    http://codepen.io/mikeflores/pen/xOZWkv?editors=0010
+    copy and pasted the code I worked and I would modify it to work with the elements in the index
```
Comment (jshawl) [inline stoplight.js:3]: please avoid copying and pasting like this in the future

**Comment by jshawl** [comment]
👍

File: script.js (hunk 1 above)
Comment (ghost) [reply script.js:29]: So I tried hyphen for `facial-hair` and it was the console kept giving me an error. :/

File: stoplight.js (hunk 5 above)
Comment (ghost) [reply stoplight.js:3]: **will not do** again. ran out of time...

**Pull request closed without merging**